src/hosting-api.js

```diff
@@ -1,27 +1,33 @@
 "use strict";
 
+import { getApiRequestHeaders } from "./helpers/index.js";
+
 /**
  * Check if a string or array of domains has been provided
  * @param {string|array} domain - The domain to check, or an array of domains to be checked.
  */
 
-function check(domain) {
+function check(domain, comment) {
```

Comment:
Would we want to add the same comment info as used in the other `checkAgainstAPI` function? u.e.

```
* @param {string} comment - Optional. The app, site, or organisation that is making the request.
```